doc/neo-tree.txt

```diff
@@ -715,6 +715,20 @@ type notations.
             winbar = false, -- toggle to show selector on winbar
             statusline = false, -- toggle to show selector on statusline
             show_scrolled_off_parent_node = false,                    -- boolean
+            sources = {                                               -- table
+                {
+                    source = "filesystem",                            -- string
+                    display_name = "  Files "                        -- string | nil
+                },
+                {
+                    source = "buffers",                               -- string
+                    display_name = "  Buffers                        -- string | nil
+                },
+                {
+                    source = "git_status",                            -- string
+                    display_name = "  Git "                          -- string | nil
+                },
+            },
             tab_labels = {                                            -- table
```

Comment:
I assume that the `tab_labels` are kept to avoid a breaking change. I would mark them as deprecated though.

Reply:
Actually I forgot to remove them! I didn't see deprecation labels in the readme so I intended to yoink this! If you would prefer I mark it as deprecated, I can do so though I don't know how exactly to do that (following the readme) as I don't see any other deprecated options in the readme.

Reply:
No, actually you're right, it should just be removed.